src/mantarray_desktop_app/mc_simulator.py

```diff
@@ -57,6 +83,7 @@ class MantarrayMCSimulator(InfiniteProcess):
         fatal_error_reporter: a queue to report fatal errors back to the main process
         testing_queue: queue used to send commands to the simulator. Should only be used in unit tests
         read_timeout_seconds: number of seconds to wait until read is of desired size before returning how ever many bytes have been read. Timeout should be set to 0 unless a non-zero value is necessary for unit testing
+        sleep_after_write_seconds: number of seconds to sleep after writing data to the simulator. This should always be None except when necessary for unit testing
```

Comment:
we don't ever want to have this sleep set to something other than `None` if we actually start the simulator right? this is just for simple unit tests when the simulator is not running, correct?

might be good in the future to override `start` and raise an error if this is set to something other than `None` (then call the `super`), so that we don't accidentally get into situations in our test cases where we have a fixture with this sleep time set, and then start it running for a more "integration"-level test.  When the simulator is running, I think we want to avoid the sleep so that it behaves as close as possible to the actual running simulator---I don't think we would need the sleep when it's running anyway